server/data/applicationClient.ts

```diff
@@ -33,6 +33,11 @@ export default class ApplicationClient {
     return (await this.restClient.get({ path: paths.applications.index.pattern })) as Array<Cas2ApplicationSummary>
   }
 
+  async getAllByPrison(prisonCode: string): Promise<Array<Cas2ApplicationSummary>> {
+    const path = `${paths.applications.index({})}?prisonCode=${prisonCode}`
```

Comment:
We also need to pass in the `isSubmitted=true` here, as we are only showing submitted applications currently (If this isn't passed in it will return all applications for the prison, which would lead to user clicking through to an in progress one and getting forbidden, because we haven't set up that user journey/permission yet)

Reply:
aha, thanks!